autoload/OmniSharp/proc.vim

```diff
@@ -52,14 +52,20 @@ function! OmniSharp#proc#vimErrHandler(channel, message) abort
   echoerr printf('%s: %s', string(a:channel), string(a:message))
 endfunction
 
+let s:job = v:null
```

Comment:
This file may be loaded in Vim that older than Vim 8.0 so we should avoid `v:null` here.

Reply:
Okay, Do you prefer to use `exists('s:job')` ?

Reply:
Sounds good 👍